src/composables/ui.ts

```diff
@@ -52,16 +53,14 @@ const otherSettings = useStorageRef<IOtherSettings>(
     migrations: [
       migrateOtherSettingsVuexToComposable,
       migrateSecureLoginEnabledToBiometric,
+      migrateRemoveSecureLoginTimeout,
     ],
   },
 );
 
 const isSeedBackedUp = computed(() => !!otherSettings.value.isSeedBackedUp);
 const saveErrorLog = computed(() => !!otherSettings.value.saveErrorLog);
 const isBiometricLoginEnabled = computed(() => !!otherSettings.value.isBiometricLoginEnabled);
-const secureLoginTimeout = computed(
```

Comment:
Had to move this to `auth` composable because `ui` composable cannot initialize `auth` composable after the rebasing to current `develop`.